docker/build_scripts/build.sh

```diff
@@ -89,5 +100,9 @@ find /opt/_internal \
   -print0 | xargs -0 rm -f
 
 for PYTHON in /opt/python/*/bin/python; do
+    # Smoke test to make sure that our Pythons work, and do indeed detect as
+    # being manylinux compatible:
     $PYTHON $MY_DIR/manylinux1-check.py
+    # Make sure that SSL cert checking works
+    $PYTHON $MY_DIR/ssl-check.py
```

Comment:
Weird: this script seems to be missing from the PR. travis is green but apparently it did not reach this part of the build script, it exited after 26min (instead of more than 30min usually) with:

```
Broadcast message from root@testing-gce-c03d1c2f-f8aa-4f73-8fd1-b3b312ac2218
	(unknown) at 1:24 ...
The system is going down for power off NOW!
```

https://travis-ci.org/pypa/manylinux/builds/124631630#L3928

It's probably a preemptible VM on GCE but I think it's a failure in travis not to report this as a failure.